Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM civisanalytics/datascience-python:6.5.1
-MAINTAINER support@civisanalytics.com
+# Default value provided here; can be overridden at build time.
+ARG DS_PYTHON_IMG_VERSION=overridden_at_build_time
```

Comment:
this is provided as a build argument when running the `docker build` command. i updated all the relevant locations in the repo where this is ran and created a script for local dev use for easy handling.

readme.md is updated with this information